esterni/verbali/verbale-esterno_2020-05-05.tex

```diff
@@ -0,0 +1,135 @@
+\documentclass{article}
+
+\input{../../commons/config}
+
+\appendToGraphicspath{../../commons/img/}
+
+\title{Verbale esterno --- 05/05/2020}
+
+\setResponsabile{Nome Cognome}
+\setRedattori{Alberto Gobbo}
+\setVerificatori{
+  Alberto Cocco
+}
+\setUso{Esterno}
+\setDescrizione{Verbale dell'incontro di GruppOne del 05/05/2020}
+\setModifiche{%
+\cellcolor{white!80!lightgray!100} & Nome Cognome & 2020--05--06 & approva documento \\%
+\cellcolor{white!80!lightgray!100} & Alberto Cocco & 2020--05--05 & verifica verbale \\%
+\multirow{-3}{*}{-} & Alberto Gobbo & 2020--05--05 & stendi verbale%
+}
+
+\disabilitaVersione{}
+\disabilitaElencoFigure{}
+\disabilitaElencoTabelle{}
+
+\begin{document}
+
+\input{../../commons/template}
+
+\section{Informazioni logistiche}%
+\label{sec:informazioni_logistiche}
+
+\begin{description}
+  \item [Luogo] chiamata Zoom
+  \item [Data] 05/05/2020
+  \item [Ora] 13:40 \symbol{8594} 14:00
+\end{description}
+
+\subsection{Membri del gruppo presenti}%
+\label{sub:membri_del_gruppo_presenti}
+
+\begin{enumerate}
+  % \item Riccardo Agatea
+  % \item Tobia Apolloni
+  % \item Riccardo Cestaro
+  % \item Alberto Cocco
+  \item Luca Ercole
+  \item Alberto Gobbo
+  \item Alessandro Rizzo
+  \item Fabio Scettro
+\end{enumerate}
+% sub:membri_del_gruppo_presenti (end)
+
+\subsection{Altri partecipanti}%
+\label{sub:altri_partecipanti}
+
+\begin{enumerate}
+  \item Prof.\ Riccardo Cardin (committente)
+\end{enumerate}
+
+\section{Introduzione}%
+\label{sec:introduzione}
+
+L'incontro, avvenuto tramite chiamata sulla piattaforma Zoom, ci è servito per chiarire i dubbi riguardanti l'allegato tecnico che dovremo presentare al prof. Riccardo Cardin in data 08/05/2020, in particolar modo allo sviluppo dei diagrammi delle classi.
```

Comment:
Al posto di "dovremo presentare" metterei presenteremo.
Girerei la frase e metterei "l'incontro, avvenuto tramite chiamata sulla piattaforma Zoom, ci è servito per chiarire i dubbi riguardanti lo sviluppo dei diagrammi delle classi da inserire nell'allegato tecnico che consegneremo al Prof.Cardin in data 08/05/2020"

Reply:
Ok correggo